src/servers/database.ts

```diff
@@ -465,7 +465,45 @@ export class DatabaseServer extends SkeletonServer {
       return null;
     }
   }
-
+  async eth_getTransactionsByBlockNumber(
+    _request: any,
+    blockNumber: web3.Quantity | web3.Tag
+  ): Promise<web3.TransactionResult | null> {
+    const blockNumber_ =
+      parseBlockSpec(blockNumber) || (await this._fetchCurrentBlockID());
+      
+    try {
+      const {
+        rows,
+      } = await this._query(
+        `SELECT
```

Comment:
The same applies here 

Would be good to move this query to the etc/schema/functions as a separate sql function.